src/main.rs

```diff
@@ -20,9 +25,97 @@ enum Opt {
         /// Verbose mode (-v, -vv, -vvv, etc.)
         #[clap(short, long, parse(from_occurrences))]
         verbose: u8,
+
+        #[clap(short, long)]
+        google_sheet: Option<String>,
     },
 }
 
+struct Collection<T: google_sheets::IntoSheetEntry> {
```

Comment:
One thing I don't love is that `Collection` feels "independent" from google sheets, and yet it mentions google sheets in its where clauses.

Reply:
This is a very good point. Will change.